### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by IvanLudick**
## In this pull requests I implemented the following:

- Add the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:
    -  "Works" section
- Make sure the content of the "About myself" section is aligned with flexbox.
- In order to lay out the elements in "Works" section you should **use CSS Grid and Flexbox** combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.
- Please use `ul` and `li` for categories in "Work" section. `Html`, `Css`, and `Javascript` should be list items.
- Please use `ul` and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.
- You don't need to implement the detail modal window that requires JavaScript.
- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

**Comment by IvanLudick** [comment]
add Work section
add About Myself section

File: index.html [hunk 1]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+      </select>
+    </div>
+
+    <div class="skill-list2">
+      <select name="Languages" id="skills">
+        <option>Skill</option>
+      </select>
+    </div>
+
+  </section>
```
Comment (MaganaAsiati) [inline index.html]: - [ ] Kindly make sure to the same typography styles (text color, font-weight, etc....) used to display the languages, languages, and skills in order to meet the provided Figma design.



## Figma design 

image

## Your design

image

File: index.html [hunk 2]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
```
Comment (MaganaAsiati) [inline index.html]: - [ ] You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works :+1:

File: index.html [hunk 3]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime2"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime2").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
```
Comment (MaganaAsiati) [inline index.html]: - You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works +1

File: index.html [hunk 4]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime3"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime3").innerHTML = dt.getFullYear();
+          </script>
```
Comment (MaganaAsiati) [inline index.html]: -  You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works +1

File: index.html [hunk 5]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime4"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime4").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
```
Comment (MaganaAsiati) [inline index.html]: -  You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works :+1:

File: index.html [hunk 6]
```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+        <ul class="social-links">
+          <li class="linkedin2"><a href="https://www.linkedin.com/in/ivan-ludick-11a055243/"><img src="images\Linkedin icon.jpg"></a></li>
+          <li class="gmail2"><a href="https://smiley20089@gmail.com"><img src="images\Gmail.jpg"></a></li>
+          <li class="github2"><a href="https://github.com/IvanLudick"><img src="images\GitHub.jpg"></a></li>
+        </ul>
+
+        <div class="resume">
+          <button class="resume-btn">Get my resume</button>
+        </div>
+      </div>
```
Comment (MaganaAsiati) [inline index.html:192]: - [ ] Kindly make sure to the same typography styles (text color, font weight, margin, padding, font family, etc, ....) for this section in order to meet the provided Figma design. :+1: 

## Figma design 

image

## your design 
image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @jossi30, I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Kindly make your design stick as much as possible to the figma design as it was reauired in the project :+1: 
- [x]  Kindly try to customize your readme file as much as possible with the available information.If a field in the readme file is not available, kindly remove it. Provide steps that users can follow to use the project on their local machines. The MIT link is also broken. Kindly fix it :+1: 
-  [x] Kindly apply the appropriate font family to your body section so that it will make your design stick to the Figma design as it was required in the project requirements :+1: 
I suggest you use the font I provide bellow ;

`@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");`

- [x] Kindly make sure the body of your website takes the max screen so that it will remove the scrolling button from left to right as it is shown in the picture below :+1: :sunglasses: 
- [x] The main container of the projects has a display: grid as it was required in the project here :ok_hand: 

> In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 5 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html [hunk 7]
```diff
@@ -28,17 +28,205 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
...
+      <p class="about-descript">Hello, I'm a software developer! I can help you build a product, feature or website look at some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      
+      <div class="social-media2">
+        <a class="socials" href="">LET'S CONNECT</a><br>
+
+        <ul class="social-links">
+          <li class="linkedin2"><a href="https://www.linkedin.com/in/ivan-ludick-11a055243/"><img src="images\Linkedin icon.jpg"></a></li>
+          <li class="gmail2"><a href="https://smiley20089@gmail.com"><img src="images\Gmail.jpg"></a></li>
+          <li class="github2"><a href="https://github.com/IvanLudick"><img src="images\GitHub.jpg"></a></li>
+        </ul>
```
Comment (Abenezer-Tilahun) [inline index.html:187]: - [ Optional ] Kindly add the remaining social icons just like the figma design.

Your Social Icons look likes this : 
  
Capture

The Figma design Look likes this : 
Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @IvanLudick 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**